pkg/issuer/acme/dns/rfc2136/rfc2136.go

```diff
@@ -155,6 +157,12 @@ func NewDNSProviderCredentials(nameserver, tsigAlgorithm, tsigKeyName, tsigSecre
 	d.tsigAlgorithm = tsigAlgorithm
 
 	d.dns01Nameservers = dns01Nameservers
+
+	glog.V(5).Infof("DNSProvider nameserver:       %s\n", d.nameserver)
+	glog.V(5).Infof("            tsigAlgorithm:    %s\n", d.tsigAlgorithm)
+	glog.V(5).Infof("            tsigKeyName:      %s\n", d.tsigKeyName)
+	glog.V(5).Infof("            tsigSecret:       %s\n", d.tsigSecret)
```

Comment:
Is it a good idea to print all this info? Seems like some of it is/may be sensitive?

Reply:
I thought about that. It seemed that because it was at `glog.V(5)`, only an admin would have access to it. Furthermore, because of the multiple levels of `base64` encoding and the cognitive load of getting it all correct before anything works, it seemed relevant to print out the actual key so it could be compared. Of course a key fingerprint would be ideal, but I don't know any standards for that in an arbitrary key like is generated here. WDYT?

Reply:
What if we put **** in the middle of the key? It's probabilistically impossible that the beginning and end of a key would match without the middle, as well that the middle of the key could be guessed from the ends. Seems to solve both problems!